app/aboot/aboot.c

```diff
@@ -5711,7 +5770,7 @@ void aboot_init(const struct app_descriptor *app)
 		}
 		else
 		{
-		if (!IS_ENABLED(ABOOT_STANDALONE))
+		if (!IS_ENABLED(ABOOT_STANDALONE)) // TODO: detect_android_from_flash()
```

Comment:
```suggestion
		if (!IS_ENABLED(ABOOT_STANDALONE))
```

I sincerely doubt anyone would care to boot mainline android on flash devices, let's avoid code churn here

Reply:
sure thing but I still sincerely doubt you'd run LOS on NAND targets like MDM modem chipsets so I don't see the need for a todo here, and I strongly prefer to have minimal changes to CAF code when possible